code/game/jobs/job/research.dm

```diff
@@ -153,11 +153,14 @@
 	mechanical_difficulty = "Medium"
 
 /datum/job/seniormentalist/equip(mob/living/carbon/human/H)
-	if (H.mind?.role_alt_title == "Senior Mentalist") //Basic level of Psionics, with ability to read others.
+	if (H.mind?.role_alt_title == "Senior Mentalist-R")
 		psi_faculties = list("[PSI_REDACTION]" = PSI_RANK_OPERANT)
-		psi_faculties = list("[PSI_COERCION]" = PSI_RANK_MASTER)
-		psi_faculties = list("[PSI_PSYCHOKINESIS]" = PSI_RANK_OPERANT)
-		psi_faculties = list("[PSI_ENERGISTICS]" = PSI_RANK_OPERANT)
+		outfit_type = /decl/hierarchy/outfit/job/science/seniormentalist/redaction
+		supervisors = "the Research Director and the Chief Medical Officer"
+	if (H.mind?.role_alt_title == "Senior Mentalist-C")
+		psi_faculties = list("[PSI_COERCION]" = PSI_RANK_OPERANT)
+		outfit_type = /decl/hierarchy/outfit/job/science/seniormentalist/coercion
+		supervisors = "the Research Director and the EZ Supervisor"
```

Comment:
either instead of an alt title, have it pick randomly between the two even if an alt title is not selected, or have the default be coercion/redaction and have only 1 alt title to change it to the other one.

Reply:
see other PR, please do not change psi_faculties in here unless you are doing a random pick between the two
changing the psi_faculties of the job will change it for anyone who joins that job later

Reply:
>changing the psi_faculties of the job will change it for anyone who joins that job later

that seems rather odd, as using alt titles was the method used by baystation. I Just tested it on a debug server and found that it properly switched from redaction to coercion after cryoing.

Reply:
@quadbium 
okay, now join it with the alt title then join it without it. you will notice an issue
i also don't like it being *restricted* behind alt titles; the default job without alt title should have psi faculties